setup.cfg

```diff
@@ -46,7 +46,7 @@ install_requires =
     pyyaml
     h5py
     uvtools
-    hera-calibration
+    hera-calibration>=3.2.2
```

Comment:
Can you explain your thought process here quickly? Newer versions of `hera_cal` have backwards compatibility for the array shapes, but shouldn't it be the other way around?

Reply:
hera_cal took out support for old array shapes semi-recently. The newest version adds support for old array shapes back in (to some parts of the code, relevant for pspec)